include/dxc/Support/dxcapi.use.h

```diff
@@ -178,6 +193,64 @@ void WriteOperationErrorsToConsole(IDxcOperationResult *pResult,
 void WriteOperationResultToConsole(IDxcOperationResult *pRewriteResult,
                                    bool outputWarnings);
 
+class DxcDllExtValidationSupport : public DxcDllSupport {
```

Comment:
We shouldn't be defining/implementing this in this header.  We should have a separate header for this, which is used by code that needs the wrapped version.  This means we also shouldn't be adding includes to this header.

Reply:
Let's create a new header `dxcapi.extval.h` next to this one, that `#include`s this one.

Then, keep the implementation of this and the wrapper interfaces to cpp files, which will be in a new library folder like: `tools/clang/lib/extval`.  This can be linked into `dxc.exe` and test dlls, while they include the new header.

Reply:
Or maybe we can just put the `.cpp` files under `lib/DxcSupport`, alongside `dxcapi.use.cpp` instead.